### DISCUSSION TITLE
10/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#120] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimento

**Comment by Pamela198713** [comment]
Ayer estuve en clases de ispc leí el material para los trabajos prácticos
Hoy me reúno para terminar la primera entrega 
El tiempo se me pasa volando y me falta más jajaja 🤣

**Comment by rpgrca** [comment]
Si hubo ayer no estuve en clase, había leído el post en el foro diciendo que no iba a haber así que ni me conecté. A la madrugada terminé con la registración desde la app a Django. Hoy voy a ver si puedo hacer el login que debería ser parecido aunque no tengo ni idea de cómo voy a hacer para sincronizar la sesión de Django con la de Android como se hacía en Angular automáticamente. A la noche me reúno con el resto del equipo para ver cómo está la parte de documentación. El sábado y domingo voy a ver qué puedo hacer de la parte web como para darle algo más de valor al sitio, y voy a ver de levantarlo otra vez en AWS.
